raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Reconsideration of Restoration Order
+(92) [1] Within days of the service of the environmental restoration order, a person who is served with an environmental restoration order may, by giving reasons in writing, request the Agency to reconsider that order; [2] Where a request has been made as provided for under subsection [1], the environmental restoration order shall continue to be effective until varied, suspended or withdrawn; [3] The Agency shall within days of receiving the request made in accordance with subsection [1], reconsider the environmental restoration order and notify in writing the person who made the request of its decision on the environmental restoration order; [4] The Agency may, after reconsidering the case, confirm, vary, suspend or withdraw the environmental restoration order; [5] The Agency may give the person who has requested a reconsideration of an environmental restoration an opportunity to be heard orally before a decision is made. 
```

Comment:
Please replace each semi-colon preceding a number in square brackets with a period.